zapisy/apps/schedulersync/scheduler_data.py

```diff
@@ -111,6 +126,21 @@ def get_logged_client():
             client.post(self.api_login_url, data=login_data)
             return client
 
+        client = get_logged_client()
+        response = client.get(self.api_config_url)
+        api_config = response.json()
+        response = client.get(self.api_task_url)
+        api_task = response.json()
+        return api_config, api_task
+
+    def lay_out_scheduler_data(self, api_config, api_task):
+        """Lays out the data fetched from Scheduler in a useful manner.
+
+        Puts a list of SZTerm in self.terms. This list contains all necessary
```

Comment:
```suggestion
        Puts a list of SchTerm in self.terms. This list contains all necessary
```
Tak samo w pozostałych miejscach gdzie ta nazwa występuje